File: 0684/sesión11 - back-end C#/SoftVen/SoftVenModel/ReportesVendedoresDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftVen.Model
{
    public class ReportesVendedoresDTO
    {
        private PeriodosDTO periodo;
        private ProductosDTO producto;
        private TiendasDTO tienda;
        private VendedoresDTO vendedor;
        private double? totalVentas;
        private double? totalDescuentos;

        public ReportesVendedoresDTO()
        {
            this.Periodo = null;
            this.Producto = null;
            this.Tienda = null;
            this.Vendedor = null;
            this.TotalVentas = null;
            this.TotalDescuentos = null;
        }

        public ReportesVendedoresDTO(PeriodosDTO periodo, ProductosDTO producto, TiendasDTO tienda, VendedoresDTO vendedor, double totalVentas, double totalDescuentos)
        {
            this.Periodo = periodo;
            this.Producto = producto;
            this.Tienda = tienda;
            this.Vendedor = vendedor;
            this.TotalVentas = totalVentas;
            this.TotalDescuentos = totalDescuentos;
        }

        public PeriodosDTO Periodo { get => periodo; set => periodo = value; }
        public ProductosDTO Producto { get => producto; set => producto = value; }
        public TiendasDTO Tienda { get => tienda; set => tienda = value; }
        public VendedoresDTO Vendedor { get => vendedor; set => vendedor = value; }
        public double? TotalVentas { get => totalVentas; set => totalVentas = value; }
        public double? TotalDescuentos { get => totalDescuentos; set => totalDescuentos = value; }
    }
}

File: 0684/sesión22 - WS - RESTFul/Back-End/ServidorRESTFul/SoftVenModel/src/main/java/pe/edu/pucp/softven/model/ReportesVendedoresDTO.java
package pe.edu.pucp.softven.model;

public class ReportesVendedoresDTO {

    private PeriodosDTO periodo;
    private TiendasDTO tienda;
    private VendedoresDTO vendedor;
    private Double totalVentas;
    private Double totalDescuentos;

    public ReportesVendedoresDTO() {
        this.periodo = null;
        this.tienda = null;
        this.vendedor = null;
        this.totalVentas = null;
        this.totalDescuentos = null;
    }

    public ReportesVendedoresDTO(PeriodosDTO periodo, ProductosDTO producto, TiendasDTO tienda, VendedoresDTO vendedor, Double totalVentas, Double totalDescuentos) {
        this.periodo = periodo;
        this.tienda = tienda;
        this.vendedor = vendedor;
        this.totalVentas = totalVentas;
        this.totalDescuentos = totalDescuentos;
    }

    public PeriodosDTO getPeriodo() {
        return periodo;
    }

    public void setPeriodo(PeriodosDTO periodo) {
        this.periodo = periodo;
    }   

    public TiendasDTO getTienda() {
        return tienda;
    }

    public void setTienda(TiendasDTO tienda) {
        this.tienda = tienda;
    }

    public VendedoresDTO getVendedor() {
        return vendedor;
    }

    public void setVendedor(VendedoresDTO vendedor) {
        this.vendedor = vendedor;
    }

    public Double getTotalVentas() {
        return totalVentas;
    }

    public void setTotalVentas(Double totalVentas) {
        this.totalVentas = totalVentas;
    }

    public Double getTotalDescuentos() {
        return totalDescuentos;
    }

    public void setTotalDescuentos(Double totalDescuentos) {
        this.totalDescuentos = totalDescuentos;
    }

}
